packages/@aws-cdk/region-info/test/__snapshots__/region-info.test.js.snap

```diff
@@ -386,7 +386,7 @@ Object {
       "application-autoscaling": "application-autoscaling.amazonaws.com",
       "autoscaling": "autoscaling.amazonaws.com",
       "codedeploy": "codedeploy.cn-northwest-1.amazonaws.com.cn",
-      "ec2": "ec2.amazonaws.com.cn",
+      "ec2": "ec2.amazonaws.com",
```

Comment:
@rix0rrr i've done as you say, just want to confirm that china regions do not need the `.cn` suffix on their service principal

Reply:
Maybe it's out of date, but this documentation seems to imply that the cn is needed https://docs.amazonaws.cn/en_us/IAM/latest/UserGuide/troubleshoot_iam-ec2.html#troubleshoot_iam-ec2_errors-info-doc

Reply:
The version with `.cn` is still valid, but it's not needed.